### ISSUE TITLE
A11y_ML .NET Model Builder_ Data_ Keyboard: Tooltip provided beside "User Column" control is not available for keyboard users

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1598]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #Keyboard; #A11yPreview;

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#)
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Recommendation" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In "Data Source Type" and select "File" radio button
7. Tab till "Browse" button and select the input file
8. Tab till "Column to predict" control and select a value and tab till "User column" and verify the tooltip available for  mouse hover over is available for keyboard only user

### Actual Result:
Tooltip provided beside "User Column" control is not available for keyboard users

### Expected Result:
Tooltip information should be available for keyboard only users as well

###User Impact:
Keyboard only users will miss the content which is given in the form of tooltip

**Comment by NiharikaTipirishetty** [comment]
@beccamc the issue is still reproducing in latest main version, could you please let us know in which version we need to verify the fix?

**Comment by beccamc** [comment]
Can you tell me the version number?

**Comment by beccamc** [comment]
@NiharikaTipirishetty, I've updated the candidate build. You can validate on  rel/candidate build 16.7.2.2141903.  Anything closed as of today can be validated on that build. Thanks!

**Comment by akashelijah05** [comment]
#Closed;
